### DISCUSSION TITLE
gongju/gson

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#172] [Announcements]
# gongju/gson

01、前世今生 我叫 Gson，是一款开源的 Java 库，主要用途为序列化 Java 对象为 JSON 字符串，或反序列化 JSON 字符串成 Java 对象。从我的名字上，就可以看得出一些端倪，我并非籍籍无名，我出身贵族，我爸就是 Google，市值富可敌国。 当然了，作为一个聪明人，我是有自知之明的，我在我爸眼里，我并不是最闪耀的那颗星。 我来到这个世上，纯属一次意外，反正我爸是这样对我说的，他总说我是从河边捡回来的，虽然我一直不太相信。对于这件事，我向我妈确认过，她听完笑得合不拢嘴，说我太天真。 长大后，我喜欢四处闯荡，因此结识了不少同行，其中就有 Jackson 和 Fastjson。

https://tobebetterjavaer.com/gongju/gson.html

**Comment by CharlesShan-hub** [comment]
写的不错
